tests/performance-test/Dockerfile

```diff
@@ -0,0 +1,24 @@
+#--- Build SAF performance test ---
+FROM golang:1.12.7
+WORKDIR /go/src/performance-test/
+
+COPY ./dashboard.go .
```

Comment:
I think this can be combined to a single line. Note a big deal.

`COPY ./dashboard.go ./main.go ./parser.go .`